### DISCUSSION TITLE
Problems sending emails.

### DISCUSSION THREAD

**Discussion by jossmanbr2022** [pocketbase/pocketbase#346] [Q&A]
1-. It is normal that it takes a long time or the verification emails are not sent.

2-. In the new update of PB when trying to use the sending of mail test mode an alert appears with the following msg:
Failed to send the test email. Raw error:
501 Invalid command or cannot parse to address.

**Comment by ganigeorgiev** [comment]
1. The email deliverability response depends on you email server, but usually it shouldn't take more than a couple of seconds.

2. Is the address you are sending the test email valid? Does it have any unusual formatting (eg. dots as prefix/suffix)? Could you test it with a SMTP configured to [mailtrap.io](https://mailtrap.io/)? 
_There are [some changes to how we build the `to` address](https://github.com/pocketbase/pocketbase/blob/master/tools/mailer/smtp.go#L69-L70) but they were tested with my local email server, mailtrap and gmail and I didn't stumble on any issues._

**Reply by ganigeorgiev** [reply]
@jossmanbr2022 Please test with v0.5.1 release. I've reverted the `to` address email change since it was causing issues with some mail servers.

A better fix for the spamassasin `TO_NO_BRKTS_HTML_ONLY` rule will be added in v0.6.0 release.

**Reply by jossmanbr2022** [reply]
Thanks for your answer, I still have problems with the delivery of the emails, I will verify with the page that you tell me that you use as a test and I will see what results I get.
